### PR TITLE
Issue 168

### PR DESCRIPTION
**Pull request by wangerekaharun**
## What is the Purpose?
Updating All Dependencies, Gradle Version, AS Version, and Kotlin Version

## What was the approach?
Used Gradle Versions Plugin to check  which dependencies have updates

## Are there any concerns to addressed further before or after merging this PR?
Yes there is one: On my setup, I cant be able to enable `viewBinding` and `dataBinding` using this approach:

```
android {
      buildFeatures {
           viewBinding = true
           dataBinding = true
       }
 }

```
## Mentions?
@michaelbukachi check on the above concern before merging this

## Issue(s) affected?
List of issues addressed by this PR.

### CODE REVIEW

File: features/about/build.gradle.kts [hunk 1]
```diff
@@ -8,7 +8,7 @@ plugins {
 android {
     compileSdkVersion(AndroidSDK.compile)
 
-    defaultConfig {
+    this.defaultConfig {
```
Comment (wangerekaharun) [inline features/about/build.gradle.kts:11]: @michaelbukachi will this be the new way to define this?

Comment (michaelbukachi) [reply features/about/build.gradle.kts:11]: @wangerekaharun oops. I was just testing something. It can be changed.

Comment (wangerekaharun) [reply features/about/build.gradle.kts:11]: Cool no worries

**Review by michaelbukachi — APPROVED** [review]

**Comment by wangerekaharun** [comment]
[Update Deps and Clean Up Gradle Files](https://trello.com/c/uQ4n1wTC/152-update-deps-and-clean-up-gradle-files)

**Pull request merged**